device/cuda/src/clusterization/clusterization_algorithm.cu

```diff
@@ -140,6 +138,12 @@ clusterization_algorithm::output_type clusterization_algorithm::operator()(
         m_gf_backup, m_adjc_backup, m_adjv_backup, m_backup_mutex.get());
     TRACCC_CUDA_ERROR_CHECK(cudaGetLastError());
 
+#ifndef NDEBUG
+    TRACCC_CUDA_ERROR_CHECK(cudaStreamSynchronize(stream));
+    assert(is_contiguous_on(measurement_module_projection(), m_mr.main, m_copy,
+                            m_stream, measurements));
+#endif
```

Comment:
This is really not an important point, but would it not be possible to express this with something like:

```c++
assert([&]() -> bool {
   TRACCC_CUDA_ERROR_CHECK(cudaStreamSynchronize(stream));
   return is_contiguous_on(measurement_module_projection(), m_mr.main, m_copy,
                            m_stream, measurements); }() == true);
```

? I'd just prefer to use `NDEBUG` directly if I can help it...